website/docs/telemetry.md

```diff
@@ -60,6 +60,10 @@ Explicit user action is required (see below) to change an `undecided` state to `
 Telemetry reporting is always disabled for [embedded FerretDB](https://pkg.go.dev/github.com/FerretDB/FerretDB/ferretdb)
 and can't be configured.
 
+:::info
+Despite the autogenerated message by `mongosh` on telemetry collection, please note that we do not share any usage data with MongoDB.
```

Comment:
maybe "Please note that no data will be shared with MongoDB Inc." would be better

Reply:
Should I remove the part about mongosh?